server/lib/src/plugins/mod.rs

```diff
@@ -343,9 +342,11 @@ impl Plugins {
         pre_cand: &[Arc<EntrySealedCommitted>],
         cand: &[EntrySealedCommitted],
     ) -> Result<(), OperationError> {
-        refint::ReferentialIntegrity::post_repl_incremental(qs, pre_cand, cand)
-            .and_then(|_| spn::Spn::post_repl_incremental(qs, pre_cand, cand))
-            .and_then(|_| memberof::MemberOf::post_repl_incremental(qs, pre_cand, cand))
+        domain::Domain::post_repl_incremental(qs, pre_cand, cand)?;
+        spn::Spn::post_repl_incremental(qs, pre_cand, cand)?;
+        refint::ReferentialIntegrity::post_repl_incremental(qs, pre_cand, cand)?;
+        spn::Spn::post_repl_incremental(qs, pre_cand, cand)?;
```

Comment:
out of curiosity, why's the spn one get run twice?

Reply:
I second this. And also, won't they all just throw "plugin {} has an unimplemented post_repl_incremental!" since they don't implement post_repls_incremental?

Reply:
Oh yeah, why is that run twice? That's a mistake.

The reason we allow it to "pass" doing nothing for now, is that after I finish the object state machine tests, I'm going to flip these to err() states and then implement them and their plugin tests on replication because there are also a bunch of states to handle there too.